### PR TITLE
prints all movies (edited pr title) [pr]

### PR DESCRIPTION
**Pull request by danMateer**
This is a test commit.

It was the best of times, it was the worst of times.

### CODE REVIEW

File: movie_loop.py [hunk 1]
```diff
@@ -8,3 +8,7 @@
   print "A great 90's movie:", film
 else:
   print "I miss the 90's"
+
+def movie_print(movies):
```
Comment (danMateer) [inline movie_loop.py]: Might be confusing to have the argument name the same as a variable name in the `for` loop.

[the remark this answers is not in the document]
Comment (danMateer) [reply movie_loop.py]: Replaying to reviewer comment.

[the remark this answers is not in the document]
Comment (danMateer) [reply movie_loop.py:12]: Added a reply.

** Edited the reply.

[the remark this answers is not in the document]
Comment (DrewMateer) [reply movie_loop.py:12]: Adding a comment from GitHub.

[the remark this answers is not in the document]
Comment (DrewMateer) [reply movie_loop.py:12]: Adding a comment in GitHub

Comment (DrewMateer) [inline movie_loop.py:12]: Starting a new conversation in GitHub.

File: movie_loop.py [hunk 2]
```diff
@@ -8,3 +8,7 @@
...
+    for movie in movies:
+        print movie
```
Comment (DrewMateer) [inline movie_loop.py:14]: new inline comment in GitHub.

**Pull request closed without merging**